dpx/src/dpx_pdfdev.rs

```diff
@@ -266,6 +276,12 @@ impl pdf_coord {
         Self { x: 0., y: 0. }
     }
 }
+impl PartialEq for pdf_coord {
```

Comment:
I think this PartialEq is very weird... Do you think make it a method is a better idea?

Reply:
More precisely, it’s symmetric but not transitive, so you shouldn’t implement PartialEq this way.

Reply:
See last commit.

Reply:
PartialEq actually doesn't have to be transitive, Eq have to be. But it's still better to not hide these kind of logic under equal operator。